rust/hyperlane-base/src/metrics/custom_metrics.rs

```diff
@@ -52,6 +66,34 @@ pub(crate) fn create_agent_metrics(metrics: &CoreMetrics) -> Result<AgentMetrics
         .build()?)
 }
 
+/// Chain-specific metrics
+#[derive(Clone, Builder)]
+pub struct ChainMetrics {
+    /// Tracks the current block height of the chain.
+    /// - `chain`: the chain name (or ID if the name is unknown) of the chain
+    ///   the block number refers to.
+    #[builder(setter(into, strip_option), default)]
+    pub block_height: Option<IntGaugeVec>,
```

Comment:
bump on this, happy to leave it optional if you feel it's useful though

Reply:
ah, misread as only `gas_price`. I removed the option from `block_height`, but since we're not implementing the gas_price median we have to keep that as optional